<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>console</class>
 <widget class="QWidget" name="console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gNAP.qrc">
    <normaloff>:/picture/picture/icon.png</normaloff>:/picture/picture/icon.png</iconset>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#background{
        border-image: url(:/picture/picture/background.jpg);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#progress_bar{
        border-image: url(:/picture/picture/progress.png);
}
#progress_bar::chunk{
        width:1px;
        background-color: rgb(255, 255, 255);

}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="cout">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>350</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cout{
        border-image: url(:/picture/picture/progress.png);
        color:white;
        font: 16pt &quot;Consolas&quot;;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyze">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>175</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#analyze{
        border-image: url(:/picture/picture/blue1.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}

#analyze:hover{

        border-image: url(:/picture/picture/blue2.png);
}</string>
    </property>
    <property name="text">
     <string>Analyse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>400</y>
      <width>175</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#show{
        border-image: url(:/picture/picture/red1.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}

#show::hover{

        border-image: url(:/picture/picture/red2.png);
}</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>175</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#result{
        border-image: url(:/picture/picture/orange1.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}

#result:hover{

        border-image: url(:/picture/picture/orange2.png);
}</string>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predict">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>400</y>
      <width>175</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#predict{
        border-image: url(:/picture/picture/green1.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}

#predict::hover{

        border-image: url(:/picture/picture/green2.png);
}</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#add{
        border-image: url(:/picture/picture/purple1.png);
        color:white;
        font: 75 18pt &quot;Arial&quot;;
}

#add:hover{

        border-image: url(:/picture/picture/purple2.png);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#close{
        border-image: url(:/picture/picture/close1.png);
}

#close:hover{

        border-image: url(:/picture/picture/close2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tips">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>700</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#tips{
        font: 75 16pt &quot;Cambria&quot;;
        color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#back{
        border-image: url(:/picture/picture/back1.png);
}

#back:hover{

        border-image: url(:/picture/picture/back2.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>171</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/picture/picture/Software-Logo_white_small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="gNAP.qrc"/>
 </resources>
 <connections/>
</ui>
